src/yz_solr.erl

```diff
@@ -279,8 +279,11 @@ prepare_json(Docs) ->
 -spec partition_list(index_name()) -> {ok, Resp::binary()} | {error, term()}.
 partition_list(Core) ->
     Params = [{q, "*:*"},
+              {rows, 0},
+              {omitHeader, <<"true">>},
               {facet, "on"},
               {"facet.mincount", "1"},
+              {"facet.method", <<"enum">>},
```

Comment:
It must not matter that some of these params are strings and others binaries. Should it be consistent?